lib/lib_rss.php

```diff
@@ -15,6 +15,8 @@ function mb_strcut($str, $start, $length = null, $encoding = 'UTF-8') {
 	openlog('FreshRSS', LOG_CONS | LOG_ODELAY | LOG_PID, LOG_USER);
 }
 
+require_once 'autoload.php';
```

Comment:
```suggestion
require_once LIB_PATH . '/autoload.php';
```

Just a suggestion to be more similar to what we do elswhere

Reply:
Fixed